packages/mc-scripts/src/config/create-webpack-config-for-development.ts

```diff
@@ -179,10 +181,9 @@ function createWebpackConfigForDevelopment(
         }),
       mergedOptions.toggleFlags.generateIndexHtml &&
         new LocalHtmlWebpackPlugin(),
-      // Strip all locales except `en`, `de`
-      // (`en` is built into Moment and can't be removed).
+      // Only keep locales that are available in the Merchant Center.
       new MomentLocalesPlugin({
-        localesToKeep: ['de', 'es', 'fr', 'zh-cn', 'ja'],
+        localesToKeep: momentLocalesToKeep,
```

Comment:
This is the result in the bundle

<img width="1361" alt="image" src="https://user-images.githubusercontent.com/1110551/177522779-b77b8fcf-c1de-4414-84a1-319a03b83d13.png">